### PR TITLE
Tokens: Add motion tokens 

### PR DESCRIPTION
**Pull request by diyorbek**
### Summary

Added motions tokens:
 - Easings
 - Durations

#### What changed?

Added custom filters and transforms for generating tokens for the each platform.

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-7923)
- [TDD](https://docs.google.com/document/d/19AaFgLFcEkBpw9UQ_xQbYZWutC9bgoPevn3PImPtwUw/edit?usp=sharing)

### Checklist

- [ ] Added unit tests
- [ ] Added documentation + accessibility tests
- [ ] Verified accessibility: keyboard & screen reader interaction
- [ ] Checked dark mode, responsiveness, and right-to-left support
- [ ] Checked stakeholder feedback (e.g. Gestalt designers, relevant feature teams)

### CODE REVIEW

**Comment by AlbertCarreras** [comment]
You need to run unit tests. 

`cd gestalt-design-token && yarn jest --updateSnapshot`

**Review by AlbertCarreras — APPROVED** [review]
LGTM! Pending Ravi

File: packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap [hunk 1]
```diff
@@ -37,12 +39,6 @@ Array [
   "dist/css/vr-theme/font-lineheight-vi.css",
   "dist/css/vr-theme/variables-dark.css",
   "dist/css/vr-theme/variables.css",
-  "dist/css/vr-theme-web-mapping/font-lineheight-ck.css",
```
Comment (rlingineni) [inline packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap:40]: Are these expected? Maybe need to rebase

File: packages/gestalt-design-tokens/tokens/vr-theme/$metadata.json [hunk 2]
```diff
@@ -11,6 +11,8 @@
     "base/rounding",
     "base/space",
     "base/color",
+    "base/motion/duration",
```
Comment (rlingineni) [inline packages/gestalt-design-tokens/tokens/vr-theme/$metadata.json:14]: This file to be updated by token studio

File: packages/gestalt-design-tokens/tokens/vr-theme/base/motion/duration.json
Comment (aarshap) [inline packages/gestalt-design-tokens/tokens/vr-theme/base/motion/duration.json:1]: I think duration.json and easing.json could be a single motion.json file at the top level of /base.

Having a lot of files has some drawbacks in Tokens Studio – happy to show this to you if you're interested.

File: packages/gestalt-design-tokens/tokens/vr-theme/$metadata.json (hunk 2 above)
Comment (AlbertCarreras) [reply packages/gestalt-design-tokens/tokens/vr-theme/$metadata.json:14]: can we check the "type"

**Review by rlingineni — APPROVED** [review]

**Pull request merged**